package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@10play/tentap-editor",
-  "version": "0.5.13",
+  "version": "0.5.14",
```

Comment:
Hey @arpitv96171 no need to update the version - after you remove this, we're good to merge just make sure the commit has a valid name

Reply:
updated